lib/net/ssh/authentication/ed25519.rb

```diff
@@ -19,18 +10,25 @@
 
 module Net; module SSH; module Authentication
 module ED25519
-  class SigningKeyFromFile < RbNaCl::Signatures::Ed25519::SigningKey
```

Comment:
This seemed... kind of crazysauce to me. Using a `SimpleDelegator` should preserve the semantics (the tests seem to be passing) while decoupling it from the internal implementation details of the `Ed25519::SigningKey` class.

Reply:
Yeah i'm not sure what was my point here either. This is not really part of the api and we can release a new version so we can get a rid of it.